modules/react-native-status/ios/RCTStatus/RCTStatus.m

```diff
@@ -227,11 +228,56 @@ - (void)handleSignal:(NSString *)signal
 ////////////////////////////////////////////////////////////////////
 #pragma mark - SendLogs method
 //////////////////////////////////////////////////////////////////// sendLogs
-RCT_EXPORT_METHOD(sendLogs:(NSString *)dbJson) {
+RCT_EXPORT_METHOD(sendLogs:(NSString *)dbJson
+                  callback:(RCTResponseSenderBlock)callback) {
     // TODO: Implement SendLogs for iOS
 #if DEBUG
     NSLog(@"SendLogs() method called, not implemented");
 #endif
+    NSFileManager *fileManager = [NSFileManager defaultManager];
+    NSError *error = nil;
+    NSURL *rootUrl =[[fileManager
+                      URLsForDirectory:NSDocumentDirectory inDomains:NSUserDomainMask]
+                     lastObject];
+    
+    NSURL *zipFile = [rootUrl URLByAppendingPathComponent:@"logs.zip"];
+    [fileManager removeItemAtPath:zipFile.path error:nil];
+    
+    NSURL *logsFolderName = [rootUrl URLByAppendingPathComponent:@"logs"];
+    
+    if (![fileManager fileExistsAtPath:logsFolderName.path])
+        [fileManager createDirectoryAtPath:logsFolderName.path withIntermediateDirectories:YES attributes:nil error:&error];
+    
+    NSURL *dbFile = [logsFolderName URLByAppendingPathComponent:@"db.json"];
+#if DEBUG
+    NSString *networkDirPath = @"ethereum/mainnet_rpc_dev";
+#else
+    NSString *networkDirPath = @"ethereum/mainnet_rpc";
+#endif
+    
+    NSURL *networkDir = [rootUrl URLByAppendingPathComponent:networkDirPath];
+    NSURL *originalGethLogsFile = [networkDir URLByAppendingPathComponent:@"geth.log"];
```

Comment:
why don't we use `ExportNodeLogs` here? https://github.com/status-im/status-go/blob/develop/mobile/status.go#L640

Reply:
Because of encoding, I wrote on Friday, forgot to remind about this on call today